### PR TITLE
docs: enable srs commands and add documentation

### PR DESCRIPTION
**Pull request by wtrocki**
This PR enables Service Registry commands to be visible by default. 
It also adds documentation for service registry.

We should not merge that before service registry is merged in UI and getting overall approval. 
I have created special prerelease for testing:

https://github.com/redhat-developer/app-services-cli/releases/tag/v0.31.0-beta1

### CODE REVIEW

**Comment by wtrocki** [comment]
Documentation needs fixes:

```
PUNCTUATION_RULE Command rhoas -> Flag - version: usage - Show rhoas version
PUNCTUATION_RULE Command rhoas cluster bind -> Flag - app-name: usage - Name of the kubernetes deployment to bind
PUNCTUATION_RULE Command rhoas cluster bind -> Flag - bind-env: usage - Bind service as environment variables
PUNCTUATION_RULE Command rhoas cluster bind -> Flag - binding-name: usage - Name of the Service binding object to create when using operator
PUNCTUATION_RULE Command rhoas cluster bind -> Flag - deployment-config: usage - Use DeploymentConfig to detect your application
PUNCTUATION_RULE Command rhoas cluster bind -> Flag - ignore-context: usage - Ignore currently selected services and ask to select each service separately
PUNCTUATION_RULE Command rhoas cluster bind -> Flag - kubeconfig: usage - Location of the kubeconfig file
PUNCTUATION_RULE Command rhoas cluster bind -> Flag - namespace: usage - Custom Kubernetes namespace (if not set current namespace will be used)
PUNCTUATION_RULE Command rhoas cluster bind -> Flag - yes: usage - Forcibly create a binding without confirmation
PUNCTUATION_RULE Command rhoas cluster connect -> Flag - ignore-context: usage - Ignore currently selected services and ask to select each service separately
PUNCTUATION_RULE Command rhoas cluster connect -> Flag - kubeconfig: usage - Location of the kubeconfig file
PUNCTUATION_RULE Command rhoas cluster connect -> Flag - namespace: usage - Custom Kubernetes namespace (if not set current namespace will be used)
PUNCTUATION_RULE Command rhoas cluster connect -> Flag - token: usage - Provide an offline token to be used by the operator (to get a token, visit https://console.redhat.com/openshift/token)

PUNCTUATION_RULE Command rhoas cluster connect -> Flag - yes: usage - Forcibly create a binding without confirmation
PUNCTUATION_RULE Command rhoas cluster status -> Flag - kubeconfig: usage - Location of the kubeconfig file
PUNCTUATION_RULE Command rhoas kafka consumer-group delete -> Flag - id: usage - The unique ID of the consumer group to delete
PUNCTUATION_RULE Command rhoas kafka consumer-group delete -> Flag - yes: usage - Skip confirmation to forcibly delete a consumer group
PUNCTUATION_RULE Command rhoas kafka consumer-group describe -> Flag - id: usage - The unique ID of the consumer group to view
PUNCTUATION_RULE Command rhoas kafka consumer-group describe -> Flag - output: usage - Format in which to display the consumer group (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka consumer-group list -> Flag - output: usage - Format in which to display the consumer group (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka consumer-group list -> Flag - page: usage - Current page number for list of consumer groups
PUNCTUATION_RULE Command rhoas kafka consumer-group list -> Flag - search: usage - Text search to filter consumer groups by ID
PUNCTUATION_RULE Command rhoas kafka consumer-group list -> Flag - size: usage - Maximum number of consumer groups to be returned per page
PUNCTUATION_RULE Command rhoas kafka consumer-group list -> Flag - topic: usage - Fetch the consumer groups for a specific Kafka topic
PUNCTUATION_RULE Command rhoas kafka consumer-group reset-offset -> Flag - id: usage - The unique ID of the consumer group to reset-offset
PUNCTUATION_RULE Command rhoas kafka consumer-group reset-offset -> Flag - offset: usage - Offset type (choose from: "earliest", "latest", "absolute", "timestamp")
PUNCTUATION_RULE Command rhoas kafka consumer-group reset-offset -> Flag - output: usage - Format in which to display reset offset result (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka consumer-group reset-offset -> Flag - partitions: usage - Partitions on which to reset the consumer group offset (comma-separated integers)
PUNCTUATION_RULE Command rhoas kafka consumer-group reset-offset -> Flag - topic: usage - Select topic for which consumer group offsets are to be reset
PUNCTUATION_RULE Command rhoas kafka consumer-group reset-offset -> Flag - value: usage - Custom offset value (required when offset is "absolute" or "timestamp")
PUNCTUATION_RULE Command rhoas kafka consumer-group reset-offset -> Flag - yes: usage - Skip confirmation to forcibly reset the offset for the consumer group
PUNCTUATION_RULE Command rhoas kafka topic create -> Flag - cleanup-policy: usage - Determines whether log messages are deleted, compacted, or both
PUNCTUATION_RULE Command rhoas kafka topic create -> Flag - name: usage - Topic name
PUNCTUATION_RULE Command rhoas kafka topic create -> Flag - output: usage - Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka topic create -> Flag - partitions: usage - The number of partitions in the topic
PUNCTUATION_RULE Command rhoas kafka topic create -> Flag - retention-bytes: usage - The maximum total size of a partition log segments before old log segments are deleted to free up space
PUNCTUATION_RULE Command rhoas kafka topic create -> Flag - retention-ms: usage - The period of time in milliseconds the broker will retain a partition log before deleting it
PUNCTUATION_RULE Command rhoas kafka topic delete -> Flag - name: usage - Topic name
PUNCTUATION_RULE Command rhoas kafka topic delete -> Flag - yes: usage - Skip confirmation to forcibly delete a topic
PUNCTUATION_RULE Command rhoas kafka topic describe -> Flag - name: usage - Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka topic describe -> Flag - output: usage - Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka topic list -> Flag - output: usage - Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka topic list -> Flag - page: usage - Current page number for list of topics
PUNCTUATION_RULE Command rhoas kafka topic list -> Flag - search: usage - Text search to filter the Kafka topics by name
PUNCTUATION_RULE Command rhoas kafka topic list -> Flag - size: usage - Maximum number of items to be returned per page
PUNCTUATION_RULE Command rhoas kafka topic update -> Flag - cleanup-policy: usage - Determines whether log messages are deleted, compacted, or both
PUNCTUATION_RULE Command rhoas kafka topic update -> Flag - name: usage - Topic name
PUNCTUATION_RULE Command rhoas kafka topic update -> Flag - output: usage - Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka topic update -> Flag - partitions: usage - The number of partitions in the topic
PUNCTUATION_RULE Command rhoas kafka topic update -> Flag - retention-bytes: usage - The maximum total size of a partition log segments before old log segments are deleted to free up space
PUNCTUATION_RULE Command rhoas kafka topic update -> Flag - retention-ms: usage - The period of time in milliseconds the broker will retain a partition log before deleting it
PUNCTUATION_RULE Command rhoas kafka create -> Flag - name: usage - Unique name of the Kafka instance
PUNCTUATION_RULE Command rhoas kafka create -> Flag - output: usage - Format in which to display the Kafka instance (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka create -> Flag - provider: usage - Cloud Provider ID
PUNCTUATION_RULE Command rhoas kafka create -> Flag - region: usage - Cloud Provider Region ID
PUNCTUATION_RULE Command rhoas kafka create -> Flag - use: usage - Set the new Kafka instance to the current instance
PUNCTUATION_RULE Command rhoas kafka create -> Flag - wait: usage - Wait until the Kafka instance is created
PUNCTUATION_RULE Command rhoas kafka delete -> Flag - id: usage - Unique ID of the Kafka instance you want to delete
PUNCTUATION_RULE Command rhoas kafka delete -> Flag - name: usage - Name of the Kafka instance you want to delete
PUNCTUATION_RULE Command rhoas kafka delete -> Flag - yes: usage - Skip confirmation to forcibly delete this Kafka instance
PUNCTUATION_RULE Command rhoas kafka describe -> Flag - bootstrap-server: usage - If specified, only bootstrap server host of the Kafka instance will be displayed
PUNCTUATION_RULE Command rhoas kafka describe -> Flag - id: usage - Unique ID of the Kafka instance you want to view
PUNCTUATION_RULE Command rhoas kafka describe -> Flag - name: usage - Name of the Kafka instance you want to view
PUNCTUATION_RULE Command rhoas kafka describe -> Flag - output: usage - Format in which to display the Kafka instance (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka list -> Flag - limit: usage - The maximum number of Kafka instances to be returned
PUNCTUATION_RULE Command rhoas kafka list -> Flag - output: usage - Format in which to display the Kafka instances (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka list -> Flag - page: usage - Display the Kafka instances from the specified page number
PUNCTUATION_RULE Command rhoas kafka list -> Flag - search: usage - Text search to filter the Kafka instances by name, owner, cloud_provider, region and status
PUNCTUATION_RULE Command rhoas kafka update -> Flag - id: usage - Unique ID of the Kafka instance you want to update
PUNCTUATION_RULE Command rhoas kafka update -> Flag - name: usage - Name of the Kafka instance you want to update
PUNCTUATION_RULE Command rhoas kafka update -> Flag - output: usage - Format in which to display the Kafka instance (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas kafka update -> Flag - owner: usage - ID of the user you want to set as the owner of this Kafka instance
PUNCTUATION_RULE Command rhoas kafka update -> Flag - yes: usage - Forcibly update the Kafka instance without confirmation
PUNCTUATION_RULE Command rhoas kafka use -> Flag - id: usage - Unique ID of the Kafka instance you want to set as the current instance
PUNCTUATION_RULE Command rhoas kafka use -> Flag - name: usage - Name of the Kafka instance you want to set as the current instance
PUNCTUATION_RULE Command rhoas service-account create -> Flag - description: usage - Description for the service account (only alphanumeric characters and '-', '.', ',' are valid)
PUNCTUATION_RULE Command rhoas service-account create -> Flag - file-format: usage - Format in which to save the service account credentials (choose from: "env", "json", "properties")
PUNCTUATION_RULE Command rhoas service-account create -> Flag - name: usage - Name of the service account
PUNCTUATION_RULE Command rhoas service-account create -> Flag - output-file: usage - Sets a custom file location to save the credentials
PUNCTUATION_RULE Command rhoas service-account create -> Flag - overwrite: usage - Forcibly overwrite a credentials file if it already exists
PUNCTUATION_RULE Command rhoas service-account delete -> Flag - id: usage - The unique ID of the service account to delete
PUNCTUATION_RULE Command rhoas service-account delete -> Flag - yes: usage - Skip confirmation to forcibly delete this service account
PUNCTUATION_RULE Command rhoas service-account describe -> Flag - id: usage - The unique ID of the service account to view
PUNCTUATION_RULE Command rhoas service-account describe -> Flag - output: usage - Format in which to display the service account (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas service-account list -> Flag - output: usage - Format in which to display the service accounts (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas service-account reset-credentials -> Flag - file-format: usage - Format in which to save the service account credentials (choose from: "env", "json", "properties")
PUNCTUATION_RULE Command rhoas service-account reset-credentials -> Flag - id: usage - The unique ID of the service account for which you want to reset the credentials
PUNCTUATION_RULE Command rhoas service-account reset-credentials -> Flag - output-file: usage - Sets a custom file location to save the credentials
PUNCTUATION_RULE Command rhoas service-account reset-credentials -> Flag - overwrite: usage - Forcibly overwrite a credentials file if it already exists
PUNCTUATION_RULE Command rhoas service-account reset-credentials -> Flag - yes: usage - Skip confirmation to forcibly reset service account credentials
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - artifact-id: usage - ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - description: usage - Custom description of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - file: usage - File location of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - name: usage - Custom name of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - output: usage - Format in which to display the Service Registry instance (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - type: usage - Type of artifact. Choose from: AVRO, PROTOBUF, JSON, OPENAPI, ASYNCAPI, GRAPHQL, KCONNECT, WSDL, XSD, XML
PUNCTUATION_RULE Command rhoas service-registry artifact create -> Flag - version: usage - Custom version of the artifact (for example 1.0.0)
PUNCTUATION_RULE Command rhoas service-registry artifact delete -> Flag - artifact-id: usage - ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact delete -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact delete -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact delete -> Flag - yes: usage - Delete artifact without prompt
PUNCTUATION_RULE Command rhoas service-registry artifact download -> Flag - content-id: usage - ContentId of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact download -> Flag - global-id: usage - Global ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact download -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact download -> Flag - hash: usage - SHA-256 hash of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact download -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact download -> Flag - output-file: usage - Location of the output file
PUNCTUATION_RULE Command rhoas service-registry artifact export -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact export -> Flag - output-file: usage - File location of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact get -> Flag - artifact-id: usage - ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact get -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact get -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact get -> Flag - output-file: usage - Location of the output file
PUNCTUATION_RULE Command rhoas service-registry artifact get -> Flag - version: usage - Version of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact import -> Flag - file: usage - File location of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact import -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact list -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact list -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact list -> Flag - limit: usage - Page limit
PUNCTUATION_RULE Command rhoas service-registry artifact list -> Flag - output: usage - Output format (json, yaml, yml)
PUNCTUATION_RULE Command rhoas service-registry artifact list -> Flag - page: usage - Page number
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-get -> Flag - artifact-id: usage - ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-get -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-get -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-get -> Flag - output: usage - Output format (json, yaml, yml)
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-set -> Flag - artifact-id: usage - ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-set -> Flag - description: usage - Custom description of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-set -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-set -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-set -> Flag - name: usage - Custom name of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact metadata-set -> Flag - output: usage - Output format (json, yaml, yml)
PUNCTUATION_RULE Command rhoas service-registry artifact state-set -> Flag - artifact-id: usage - ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact state-set -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact state-set -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact state-set -> Flag - state: usage - new artifact state
PUNCTUATION_RULE Command rhoas service-registry artifact update -> Flag - artifact-id: usage - ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact update -> Flag - description: usage - Custom description of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact update -> Flag - file: usage - File location of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact update -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact update -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact update -> Flag - name: usage - Custom name of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact update -> Flag - output: usage - Format in which to display the Service Registry instance (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas service-registry artifact update -> Flag - version: usage - Custom version of the artifact (for example 1.0.0)
PUNCTUATION_RULE Command rhoas service-registry artifact versions -> Flag - artifact-id: usage - ID of the artifact
PUNCTUATION_RULE Command rhoas service-registry artifact versions -> Flag - group: usage - Artifact group
PUNCTUATION_RULE Command rhoas service-registry artifact versions -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry artifact versions -> Flag - output: usage - Output format (json, yaml, yml)
PUNCTUATION_RULE Command rhoas service-registry role add -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry role add -> Flag - role: usage - Role to apply
PUNCTUATION_RULE Command rhoas service-registry role add -> Flag - service-account: usage - ServiceAccount name
PUNCTUATION_RULE Command rhoas service-registry role add -> Flag - username: usage - Username of the user within organization
PUNCTUATION_RULE Command rhoas service-registry role list -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry role list -> Flag - output: usage - Output format (json, yaml, yml)
PUNCTUATION_RULE Command rhoas service-registry role revoke -> Flag - instance-id: usage - ID of the Service Registry instance to be used. By default, uses the currently selected instance
PUNCTUATION_RULE Command rhoas service-registry role revoke -> Flag - service-account: usage - ServiceAccount name
PUNCTUATION_RULE Command rhoas service-registry role revoke -> Flag - username: usage - Username of the user within organization
PUNCTUATION_RULE Command rhoas service-registry create -> Flag - name: usage - Unique name of the Service Registry instance
PUNCTUATION_RULE Command rhoas service-registry create -> Flag - output: usage - Format in which to display the Service Registry instance (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas service-registry create -> Flag - use: usage - Set the new Service Registry instance to the current instance
PUNCTUATION_RULE Command rhoas service-registry delete -> Flag - id: usage - Unique ID of the Service Registry instance you want to delete (if not provided, the current Service Registry instance will be deleted)
PUNCTUATION_RULE Command rhoas service-registry delete -> Flag - name: usage - Name of the Service Registry instance to delete
PUNCTUATION_RULE Command rhoas service-registry delete -> Flag - yes: usage - Skip confirmation to forcibly delete this Service Registry instance
PUNCTUATION_RULE Command rhoas service-registry describe -> Flag - id: usage - Unique ID of the Service Registry instance (if not provided, the current Service Registry instance will be used)
PUNCTUATION_RULE Command rhoas service-registry describe -> Flag - name: usage - Name of the Service Registry instance to view
PUNCTUATION_RULE Command rhoas service-registry describe -> Flag - output: usage - Format in which to display the Service Registry instance (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas service-registry list -> Flag - limit: usage - The maximum number of Service Registry instances to be returned
PUNCTUATION_RULE Command rhoas service-registry list -> Flag - output: usage - Format in which to display the Service Registry instance (choose from: "json", "yml", "yaml")
PUNCTUATION_RULE Command rhoas service-registry list -> Flag - page: usage - Display the Service Registry instances from the specified page number
PUNCTUATION_RULE Command rhoas service-registry use -> Flag - id: usage - Unique ID of the Service Registry instance you want to set as the current instance
PUNCTUATION_RULE Command rhoas service-registry use -> Flag - name: usage - Name the Service Registry instance you want to set as the current instance
PUNCTUATION_RULE Command rhoas login -> Flag - api-gateway: usage - URL of the API gateway
PUNCTUATION_RULE Command rhoas login -> Flag - auth-url: usage - The URL of the SSO Authentication server
PUNCTUATION_RULE Command rhoas login -> Flag - client-id: usage - OpenID client identifier
PUNCTUATION_RULE Command rhoas login -> Flag - insecure: usage - Enables insecure communication with the server by disabling TLS certificate and host name verification
PUNCTUATION_RULE Command rhoas login -> Flag - mas-auth-url: usage - The URL of the identity.api.openshift.com Authentication server
PUNCTUATION_RULE Command rhoas login -> Flag - print-sso-url: usage - Prints the console login URL, which you can use to log in to RHOAS from a different web browser (this is useful if you need to log in with different credentials than the credentials you used in your default web browser)
PUNCTUATION_RULE Command rhoas login -> Flag - scope: usage - Override the default OpenID scope (to specify multiple scopes, use a separate --scope for each scope)
PUNCTUATION_RULE Command rhoas login -> Flag - token: usage - Log in using an offline token, which can be obtained at https://console.redhat.com/openshift/token
PUNCTUATION_RULE Command rhoas status -> Flag - output: usage - Format in which to display the status of your services (choose from: "json", "yml", "yaml")
```

File: docs/commands/rhoas_service-registry_artifact.adoc [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+Service Registry enables developers to manage and share the structure of their data.
+For example, client applications can dynamically push or pull the latest schema or API updates to or from the registry without needing to redeploy.
+Service Registry also enables developers to create rules that govern how registry content can evolve over time.
+For example, this includes rules for content validation and version compatibility.
+
+Artifact commands enable client applications to manage the schema and API artifacts in the registry instance.
+This set of commands provide create, read, update, and delete operations for artifacts, rules, versions, and metadata.
+
+This command is available as part of the developer preview
```
Comment (smccarthy-ie) [inline docs/commands/rhoas_service-registry_artifact.adoc:23]: Suggest the following (occurs in multiple files):
This command is available as part of the Development Preview release.

File: docs/commands/rhoas_service-registry_role.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+
+Service Registry role management
+Set of commands that give users one of following permissions:
+
+- READ_ONLY (read artifacts)
+- DEVELOPER (write access to all resources)
+- ADMIN (Export/Import artifacts, Manage Roles)
+
+Roles can be applied to users and service accounts generated by cli (service account ids
```
Comment (smccarthy-ie) [inline docs/commands/rhoas_service-registry_role.adoc:19]: Suggest: 
Roles can be applied to users and service accounts generated by the CLI (service account IDs).

These commands are only accessible to users with the organization admin role or owners of the Service Registry instance.

This command is available as part of the Development Preview release.

**Review by rkpattnaik780 — CHANGES_REQUESTED** [review]
Currently the `rhoas status` command doesn't show status of service-registry by default. It should be available already.

**Comment by wtrocki** [comment]
Good guy charmil saved us tons of verification again! Thx @ankithans

**Review by rkpattnaik780 — APPROVED** [review]

**Pull request merged**